lib/elements/dom-module.html

```diff
@@ -74,6 +74,10 @@
       return null;
     }
 
+    /**
+     * @param {string} name
```

Comment:
`@param {?string} old`
`@param {?string} value`

Reply:
Done. Also had to add descriptions according to linter.